aws-sqs: add page (#9085)
tldr-pages/tldr: pages/common/aws-sqs.md

# aws sqs

> > CLI for AWS SQS -  provides simple queue for storing messages to share it between applications.

Reviewer: You're describing the SQS services instead of the command itself.
```suggestion
> Create, delete, and send messages to queues for the AWS SQS service.
```

> More information: <https://docs.aws.amazon.com/cli/latest/reference/sqs/>.

- List all availables queues:

`aws sqs list-queues`

- Get a queue url:

`aws sqs get-queue-url --queue-name {{queue_name}}`

- Create queue with specified name:

`aws sqs create-queue --queue-name {{queue_name}} --attributes file://{{queue_atributes_file.json}}`

- Send message to queue:

`aws sqs send-message --queue-url {{https://sqs.us-east-1.amazonaws.com/my_queue}} --message-body "{{My message body}}" --delay-seconds {{##}} --message-attributes file://{{my-message.json}}`

- Delete specified message from queue:

> `aws sqs delete-message --queue-url {{https://queue_url}} --receipt-handle {{message_receipt-handle}}`

Reviewer: ```suggestion
`aws sqs delete-message --queue-url {{https://queue_url}} --receipt-handle {{receipt_handle}}`
```

> - Delete specific queue:

Reviewer: ```suggestion
- Delete a specific queue:
```

> `aws sqs delete-queue --queue-url {{https://sqs.us-east-1.amazonaws.com/my_queue}}`

Reviewer: ```suggestion
`aws sqs delete-queue --queue-url https://sqs.{{region}}.amazonaws.com/{{queue_name}}`
```

> - Delete All messages in specified queue:

Reviewer: ```suggestion
- Delete all messages from the specified queue:
```

> `aws sqs purge-queue --queue-url {{https://sqs.us-east-1.amazonaws.com/my_queue}}`

Reviewer: ```suggestion
`aws sqs purge-queue --queue-url https://sqs.{{region}}.amazonaws.com/{{queue_name}}`
```

> - Enable AWS account to send messages to queue:

Reviewer: ```suggestion
- Enable a specific AWS account to send messages to queue:
```

> `aws sqs add-permission --queue-url {{https://sqs.us-east-1.amazonaws.com/my_queue}} --label {{sendMessagesPermission}} --aws-account-ids {{AWS-ACCOUNT-ID}} --actions SendMessage`

Reviewer: Based on the man page
> --label (string)
> 
>     The unique identification of the permission you’re setting (for example, AliceSendMessage ). 
So `permission_name` fits better
```suggestion
`aws sqs add-permission --queue-url https://sqs.{{region}}.amazonaws.com/{{queue_name}} --label {{permission_name}} --aws-account-ids {{account_id}} --actions SendMessage`
```
